models/chain_swap/arbitrum/chain_swap_arbitrum_trades.sql

```diff
@@ -0,0 +1,143 @@
+{{
+    config(
+        alias="trades",
```

Comment:
Please use `'` instead of `"` in the spellbook. As `"` may refer to column name.
```suggestion
        alias='trades',
```